app/views/users/sessions/login_email_sent.html.erb

```diff
@@ -1,6 +1,6 @@
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
-    <h2 class="govuk-heading-l">Check your email</h2>
+    <h1 class="govuk-heading-l">Check your email</h1>
```

Comment:
I always wanted to sort out our header mess, potentially there is still some, but slightly less now :)